apps/web/src/server/lib/nunjucks-template-builders/summary-list-builder.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * @param {any} header
+ * @param {any[]} rowArray
+ * @returns {object}
```

Comment:
We could be more specific with this return type, and all the types actually - the code assumes a particular type.